database/seeds/DatabaseSeeder.php

```diff
@@ -21,7 +21,8 @@ public function run()
             ProcessSystemCategorySeeder::class,
             GroupSeeder::class,
             ScreenTypeSeeder::class,
-            ScreenSystemSeeder::class
+            ScreenSystemSeeder::class,
```

Comment:
Do we need all of these seeders with this change?

Reply:
The default screen is used in Feature interstitial